Source/DafnyCore/Reporting.cs

```diff
@@ -242,7 +236,7 @@ public class ConsoleErrorReporter : BatchErrorReporter {
           errorLine += $" {msg} {TokenToString(tok)}";
         }
 
-        if (DafnyOptions.O.CompileVerbose) {
+        if (DafnyOptions.O.CompileVerbose && false) { // Need to control tests better before we enable this
```

Comment:
Did you intend to keep this in production? looks like a leftover 🍕

Reply:
I wanted to put out error detail when --verbose was true. But many tests have --verbose (equivalently /compileVerbose) true. But you are correct that I forgot about this.

Reply:
Took out the block for now.